ui/webpack.config.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+
+// check/detect if a production build (or not) via the webpack -p flag
+const IS_PRODUCTION = process.argv.indexOf('-p') != -1;
+
+const parentDir = path.join(__dirname, './');
+const PUBLIC_DIR = path.resolve(__dirname, './public/');
```

Comment:
Just to confirm - public is inside `src` correct?

Reply:
Nope, they both exist as a child to `ui`